src/cctz_info.cc

```diff
@@ -626,9 +630,22 @@ bool TimeZoneInfo::Load(const std::string& name) {
     path += name;
   }
 
+#ifdef _WIN32 || _WIN64
+  // Switching Linux-path to Windows-path
+  size_t pos = 0;
+  while ((pos = path.find("/", pos)) != std::string::npos) {
+    path.replace(pos, 1, "\\");
+    pos += 2;
+  }
+#endif // _WIN32 || _WIN64
+
   // Load the time-zone data.
   bool loaded = false;
+#ifdef _WIN32 || _WIN64
+  if (FILE* fp = fopen(path.c_str(), "rb")) {
```

Comment:
Can't "rb" be used as the open mode regardless of platform to avoid cluttering this part with the #ifdef?